pkg/virt-controller/services/template.go

```diff
@@ -1304,7 +1304,7 @@ func checkForKeepLauncherAfterFailure(vmi *v1.VirtualMachineInstance) bool {
 }
 
 func (t *templateService) VMIResourcePredicates(vmi *v1.VirtualMachineInstance, networkToResourceMap map[string]string) VMIResourcePredicates {
-	memoryOverhead := GetMemoryOverhead(vmi, t.clusterConfig.GetClusterCPUArch())
+	memoryOverhead := GetMemoryOverhead(vmi, t.clusterConfig.GetClusterCPUArch(), t.clusterConfig.GetConfig().GuestToRequestMemoryHeadroomRatio)
```

Comment:
I wish there was an easy way to simply get the config from within the GetMemoryOverhead function...

Reply:
Yeah, it was a bit painful to write..
I'm not sure how we can improve this in a straight forward way, as this function is being called from different containers. We can always fetch Kubevirt with an API call, but that would be costly in terms of performance.